app/mobile-content/media/media-gallery/media-gallery.component.html

```diff
@@ -27,35 +27,37 @@
   </button>
 </div>
 
-<md-grid-list cols="2" rowHeight="4:1.5" gutterSize="30px">
-  <md-grid-tile *ngFor="let medium of media | paginate: { id: 'server',
+<md-list>
+  <md-list-item (click)="selectMedium(medium)"  [ngClass]="{'clickable': selectMode}"
+                *ngFor="let medium of media | paginate: { id: 'server',
                                                           itemsPerPage: pageSize,
                                                           currentPage: currentPage,
                                                           totalItems: totalItems }">
-    <div class="tile-content">
-      <div class="media-title">
-        {{ medium.type | lowercase | translate }}:
-        {{ medium.title }}
-        ({{ medium.status | translate}})
-      </div>
-      <div *ngFor="let preview of previewImages">
-        <img class="image-size" *ngIf="medium.id === preview.id" [src]="preview.url" alt="{{ 'image preview' | translate }}">
-      </div>
-      <div class="ellipsis">
-        <div class="description">{{ medium.description }}</div>
-      </div>
-      <div *ngIf="!selectMode">
-        <button md-icon-button color="primary" (click)="editMedium(medium)" title="{{ 'edit' | translate }}">
-          <md-icon>edit</md-icon>
-        </button>
-        <button md-icon-button color="warn" (click)="deleteMedium(medium)" *ngIf="!medium.used" title="{{ 'delete' | translate }}">
-          <md-icon>delete_forever</md-icon>
-        </button>
-      </div>
-      <button md-icon-button *ngIf="selectMode" color="primary" (click)="selectMedium(medium)" title="{{ 'select' | translate }}">
-        <md-icon>done</md-icon>
-      </button>
-    </div>
-  </md-grid-tile>
-</md-grid-list>
+
+
+    <md-icon md-list-icon class="type-icon" *ngIf="!previews.has(medium.id)">
+      {{ medium.isAudio() ? 'audiotrack' : 'camera' }}
+    </md-icon>
+    <img md-list-avatar *ngIf="previews.has(medium.id)" [src]="previews.get(medium.id)" alt="{{ 'image preview' | translate }}">
+
+    <h3 md-line>{{ medium.title }}</h3>
+
+    <p md-line>
+      <span>{{ medium.type | lowercase | translate }}</span>
+      &nbsp;&bull;&nbsp;
+      <span [ngClass]="{'warning': !medium.isPublished()}">{{ medium.status | translate }}</span>
+      &nbsp;&bull;&nbsp;
+      <span [ngClass]="{'warning': !medium.used}">{{ (medium.used ? 'used' : 'unused') | translate }}</span>
+    </p>
+
+    <p md-line>{{ medium.description }}</p>
```

Comment:
Compared to the old solution, this one will show the whole description for each media item, correct? So if the description gets really long, the layout of this page might suffer. Do we maybe want to use a cutoff similar to the what it has been with the grid-list?

Reply:
Nope. This is why I threw out previous code that automatically truncated long strings. With md-list, Material will do it for us. I used the same thing for mobile pages and the text gets truncated with an ellipsis automatically.

Reply:
Ok, good. Wasn't aware that that's what md-list does.